src/pages/guide/language/index.md

```diff
@@ -15,11 +15,11 @@ Reason is an umbrella project that provides a curated layer for [OCaml](http://o
 
 #### Worry-free
 
-Reason leverages OCaml's language design and type system to give immediate feedback in the form of compile-time errors and avoided anti-patterns.
+Reason leverages OCaml's language design and type system to give immediate feedback in the form of compile-time errors and language features that make it easy to avoid anti-patterns.
 
 #### Low Overhead
 
-Reason keeps things simple and lean. Thanks to this, integrating Reason into your existing JavaScript/OCaml project is [hassle-free](./gettingStarted.html#javascript-workflow). Do more with less; reduce your future burden!
+Reason keeps things simple and lean. Thanks to this, integrating Reason into your existing [JavaScript](./getting-starded.md#javascript-workflow) or [OCaml](./getting-starded.md#native-workflow) project is hassle-free. Do more with less; reduce your future burden!
```

Comment:
THis was linking to the JavaScript workflow, even though it was referring to both JavaScript and OCaml projects. This change makes it so it links to each workflow page respectively.

These links don't actually seem to be working at the moment though (even on the deployed site) so that's a separate issue that needs addressing.